.travis.yml

```diff
@@ -103,7 +103,15 @@ matrix:
         - CLANG_CXX=clang++
         - LLVM_CONFIG=llvm-config
         - DEPLOY=1
-
```

Comment:
Nit: don't remove newline here :P

As a general rule, try to avoid unrelated whitespace changes (don't add/remove newlines in parts of the code you're not modifying), as they make the vcs history needlessly more complex.

This is OK though, don't worry about it (when a review comment starts with "Nit:" it means that it is a minor comment that can be also be ignored).

Reply:
Will remember that, thanks a lot 😄